src/frontend/src/components/warnBox.ts

```diff
@@ -0,0 +1,36 @@
+import { html, TemplateResult } from "lit-html";
```

Comment:
note about the file: this is a pretty straightforward component but for good measure would still be nice to have a line or two (short) lines about what this is and how to use it.